feature_generator.py

```diff
@@ -70,8 +70,10 @@ def __init__(self, mutating_pos_feat=None, neighbors_feat_old=None, neighbors_fe
         """
         if mutating_pos_feat is not None:
             self.mutating_pos_feats = np.array([mutating_pos_feat], dtype=int)
+            self.mutating_pos = np.array([mutating_pos], dtype=int)
```

Comment:
where is `mutating_pos` used?

Reply:
`_get_risk_update` uses it, e.g., here

https://github.com/matsengrp/samm/blob/7b9fbe9f38b5d3a209016f152b47db742362df1e/mutation_order_gibbs.py#L348

It's used when we update the risk vector so that, if a position has just mutated, it won't contribute to the subsequent risks.